src/service/topics.ts

```diff
@@ -99,34 +99,32 @@ const getWhoToFollowList = async (topics, locations, page = 1, axiosOptions = {}
 };
 
 export type TopicLatestPostData = {
-  anonymous: boolean;
-  author_user_id: string;
-  created_at: string;
-  getstream_activity_id: string;
-  is_anonymous: boolean;
+  success?: boolean;
+  status?: string;
   message: string;
-  post_content: string;
-  post_id: string;
-  topic_id: string;
-  updated_at: string;
-  username: string;
-  visibility_location_id: string;
   anon_user_info_color_code?: string;
   anon_user_info_color_name?: string;
   anon_user_info_emoji_code?: string;
   anon_user_info_emoji_name?: string;
-  audience_id?: string;
-  duration?: string;
-  parent_post_id?: string;
-  profile_pic_path?: string;
+  anonimity?: boolean;
+  created_at?: string;
+  duration_feed?: string;
+  expired_at?: string;
+  id?: string;
+  unread_count?: number;
+  time?: string;
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `TopicLatestPostData` type has been significantly changed. It's important to ensure that these changes are reflected wherever this type is used in the codebase to avoid potential type errors or unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->